src/cljs/harja/tiedot/urakka/toteumat/maarien_toteumat.cljs

```diff
@@ -124,6 +124,9 @@
                                   :else ""))
          parametrit {:polku :tehtavat
                      :filtteri (case tyyppi
+                                 ;; TODO: saako tähän jonkin filtterin sitä varten, ettei pystyis kirjaamaan määrämitattavia äkillisille ja korjauksille
+                                 ;; :maaramitattava
```

Comment:
Minusta tuntuu, että tämä ei ole oikea paikka rajoittaa sitä, mitä tehtäviä tässä haetaan, vaan se pitäisi olla siinä tietokantahaussa, jota tämä käyttää. Mutta, vaikea ottaa tästä koppia näin githubissa.